### PR TITLE
New list_translators command to list translators as ini

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

**Comment by nijel** [comment]
Thanks for your patch, can you please:

* Fix the code for Python 3 (see failure at https://travis-ci.org/nijel/weblate/jobs/145931807)
* Document the command in Management commands section (docs/admin/management.rst)
* Fix lint errors (short variable name, indentation, blank lines) (see https://landscape.io/diff/408502)

**Comment by roidelapluie** [comment]
updated

File: docs/admin/management.rst [hunk 1]
```diff
@@ -454,3 +454,21 @@ Fetches remote VCS repositories and updates internal cache.
...
+
+    [French]
+    Jean Dupont <jean.dupont@example.com>
+    [English]
+    John Doe <jd@exemple.com>
+
+You can use the --code version to use language code instead of language name.
+
+You can either define which project or component to update (eg.
+``weblate/master``) or use ``--all`` to update all existing components.
```
Comment (nijel) [inline docs/admin/management.rst]: I guess it should list, not update here :-).

File: docs/faq.rst [hunk 2]
```diff
@@ -396,6 +396,10 @@ standard VCS tools you use for code.
 Additionally, when translation file format supports it, the file headers are
 updated to include translator name.
 
+A command is also available to get translators list in an ini format::
```
Comment (nijel) [inline docs/faq.rst]: Add ```(see :djadmin:`list_translators`)``` to link to complete documentation for the command.

**Comment by roidelapluie** [comment]
Updated :)

**Comment by nijel** [comment]
Thanks, merging now...

**Pull request merged**